.travis.yml

```diff
@@ -52,11 +53,12 @@ script:
  - git diff --exit-code
  - mkdir -p build
  - cd build
- - py.test --cov-report xml --cov=pymtl3 ../pymtl3 ../examples $CONFIG --hypothesis-profile CI
+ - export NOT_USING_PYPY=$(python -V | grep -q "PyPy"; echo $?)
```

Comment:
@cbatten @ptpan This is how I turn off coverage for pypy. Looks kinda hacky. Feedbacks are welcome.